test/e2e/autoscale_test.go

```diff
@@ -129,8 +157,73 @@ func (suite *AutoscaleTestSuite) TestAutoScaleCollector() {
 	require.Greater(t, maxCollectorCount, 1, "Collector never scaled")
 }
 
+func getSimpleStreaming(name, namespace string) *v1.Jaeger {
+	kafkaClusterURL := fmt.Sprintf("my-cluster-kafka-brokers.%s:9092", kafkaNamespace) // FIXME this may need to change
+	ingressEnabled := true
+	collectorOptions := make(map[string]interface{})
+	collectorOptions["kafka.producer.topic"] = "jaeger-spans"
+	collectorOptions["kafka.producer.brokers"] = kafkaClusterURL
+	collectorOptions["kafka.producer.batch-linger"] = "1s"
+	collectorOptions["kafka.producer.batch-size"] = "128000"
+	collectorOptions["kafka.producer.batch-max-messages"] = "100"
+
+	autoscale := true
+	var minReplicas int32 = 1
+	var maxReplicas int32 = 5
+
+	jaeger := &v1.Jaeger{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Jaeger",
+			APIVersion: "jaegertracing.io/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: v1.JaegerSpec{
+			Strategy: v1.DeploymentStrategyStreaming,
+			Collector: v1.JaegerCollectorSpec{
+				Options: v1.NewOptions(collectorOptions),
+			},
+			Ingester: v1.JaegerIngesterSpec{
+				JaegerCommonSpec: v1.JaegerCommonSpec{
+					Resources: corev1.ResourceRequirements{
+						Limits: corev1.ResourceList{
+							corev1.ResourceCPU:    resource.MustParse(cpuResourceLimit), // FIXME this might need to change
+							corev1.ResourceMemory: resource.MustParse(memoryResourceLimit),
+						},
+					},
+				},
+				Options: v1.NewOptions(map[string]interface{}{
+					"kafka.consumer.topic":      "jaeger-spans",
+					"kafka.consumer.brokers":    kafkaClusterURL,
+					"ingester.deadlockInterval": "0",
+					"ingester.parallelism":      "6900",
```

Comment:
That's a very specific number :-) What's the story behind it?

Reply:
Is an attempt to saturate the resources of the ingester pod, I saw that is easier if we have high parallelism.

Reply:
You'll have to ask @rubenvp8510

Reply:
> Is an attempt to saturate the resources of the ingester pod, I saw that is easier if we have high parallelism.

Sounds reasonable, but why "6900" and not, say, "7000", or even "10000" :-)